src/ontology/edits/oeo-physical.omn

```diff
@@ -3733,6 +3737,248 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/768",
+    
+    SubClassOf: 
+        OEO_00000218
+    
+    
+Class: OEO_00010098
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Marine current energy transformation is a hydroelectric energy transformation that converts marine current energy to electrical energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine current energy transformation"@en
+    
+    SubClassOf: 
+        OEO_00110005,
+        <http://purl.obolibrary.org/obo/RO_0000057> some OEO_00010105,
+        <http://purl.obolibrary.org/obo/RO_0002233> some OEO_00010107
+    
+    
+Class: OEO_00010099
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Marine tidal energy transformation is a hydroelectric energy transformation that converts kinetic energy from tidal flow to electrical energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine tidal energy transformation"@en
+    
+    SubClassOf: 
+        OEO_00110005,
+        <http://purl.obolibrary.org/obo/RO_0000057> some OEO_00010105,
+        <http://purl.obolibrary.org/obo/RO_0002233> some OEO_00010101
+    
+    
+Class: OEO_00010100
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Marine tidal energy is the hydro energy of a tidal flow.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine tidal energy"@en
+    
+    SubClassOf: 
+        OEO_00000218
+    
+    
+Class: OEO_00010101
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Tidal flow is a water flow during which movements of water masses caused by varying gravitational and rotational forces from sun and moon, combined with the rotation of the earth, cause waters to undergo periodic depth oscillations (tides).",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "tidal flow"@en,
+        owl:equivalentClass "http://purl.obolibrary.org/obo/ENVO_01001342"
+    
+    SubClassOf: 
+        OEO_00110002
+    
+    
+Class: OEO_00010102
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine wave energy"@en,
+        owl:equivalentClass "Marine current energy is the hydro energy of moving marine water."
```

Comment:
The definition should be _Marine current energy is the hydro energy of **a tidal flow**._
Also, use used `owl:equivalentClass` instead of `definition` here.

Reply:
I got this wrong the first time: We are talking about `marine **wave** energy` here, so the definition should of course be _Marine **wave** energy is ..._

Reply:
Now this should be fixed.